graphql/types/user.ts

```diff
@@ -25,4 +25,10 @@ export class UserType implements User {
     student?: Student;
     @Field({ nullable: true })
     screener?: Screener;
+
+    @Field(() => Int, { nullable: true })
```

Comment:
I don't think this is needed for FieldResolvers - @ Field is for autogenerated getters.